spec/request/security_groups_spec.rb

```diff
@@ -701,15 +705,6 @@
         security_group_3.update(staging_default: true)
       end
 
-      it 'filters on guids' do
```

Comment:
Is there a reason we've chosen to get rid of this it block? `guids` is [still a supported filter to use](https://v3-apidocs.cloudfoundry.org/version/3.101.0/index.html#list-security-groups), and we have testing blocks for the other filters, so it seems beneficial to still have this coverage. Let me know if I'm overlooking something!

Reply:
@mkocher @mariash just giving this a bump, other than the EasyCLA this is the only outlying thing on this PR

Reply:
Agreed, not sure how that happened. I've put it back.